### ISSUE TITLE
error when use it on windows emacs 24.4

### ISSUE THREAD

**Issue by shelper** [paetzke/py-yapf.el#2]
i have the 
`Error: (error "invalid rcs patch or internal error in py-yapf-bf--apply-rcs-patch")`
when run `py-yapf-buffer` on any python file 
i am using windows and emacs 24.4

i checked the source code of py-yapf.el but could not figure out why :(

**Comment by paetzke** [comment]
Hi.

When you run `py-yapf-buffer` there should be a `*yapf Errors*` and a `*yapf patch*` buffer. Could you provide the content of these buffers?

**Comment by shelper** [comment]
the _yapf path_ gives the message 

> cygwin warning:
> MS-DOS style path detected: c:/Users/zyuan/AppData/Local/Temp/yapf11008fyG.py
>       Preferred POSIX equivalent is: /cygdrive/c/Users/zyuan/AppData/Local/Temp/yapf11008fyG.py
>       CYGWIN environment variable option "nodosfilewarning" turns off this warning.
>       Consult the user's guide for more details about POSIX paths:
>       http://cygwin.com/cygwin-ug-net/using.html#using-pathnames"

the _yapf patch_ is empty

the _backtrace_ gives message:

> Debugger entered--Lisp error: (error "invalid rcs patch or internal error in py-yapf-bf--apply-rcs-patch")
>   signal(error ("invalid rcs patch or internal error in py-yapf-bf--apply-rcs-patch"))
>   error("invalid rcs patch or internal error in py-yapf-bf--apply-rcs-patch")
>   py-yapf-bf--apply-rcs-patch(#<buffer *yapf patch*>)
>   py-yapf-bf--apply-executable-to-buffer("yapf" py-yapf--call-executable nil "py")
>   py-yapf--call()
>   py-yapf-buffer()
>   run-hooks(before-save-hook)
>   #[0 "\300\301!\207" [run-hooks before-save-hook] 2 "\n\n(fn)"]()
>   funcall(#[0 "\300\301!\207" [run-hooks before-save-hook] 2 "\n\n(fn)"])
>   basic-save-buffer()
>   save-buffer(1)
>   call-interactively(save-buffer nil nil)
>   command-execute(save-buffer)

**Comment by paetzke** [comment]
I've tried to reproduce it: I have a fresh Windows 7-64, I installed cygwin (including diffutils "Searching for program: no such file or directory, diff"). I installed Python3 (https://www.python.org/ftp/python/3.4.3/python-3.4.3.amd64.msi) and Emacs (https://ftp.gnu.org/gnu/emacs/windows/emacs-24.3-bin-i386.zip). And I installed yapf via pip.

It worked. I have no further experience with Windows. This was my first time installing "unix" tools. That are differences to your setup?
